res/css/views/elements/_CopyableText.scss

```diff
@@ -46,3 +41,11 @@ limitations under the License.
         }
     }
 }
+
+.mx_CopyableText_border {
+    border-radius: 5px;
+    border: solid 1px $light-fg-color;
+    margin-bottom: 10px;
+    margin-top: 10px;
+    padding: 10px;
+}
```

Comment:
NIT: I think it could be nicer to nest this like `&.mx_CopyableText_border` but that's just a personal preference

Reply:
I thought it would be more consistent too, but when it's nested the border disappears where it should be applied, eg: 
![image](https://user-images.githubusercontent.com/94136530/163133494-15d0376b-7fd9-4e37-90d3-aa7a9b711cb0.png)

Reply:
You probably missed the leading `&` @olivialivia

Reply:
Yes, I have. All fixed now.